README.md

```diff
@@ -73,56 +62,44 @@ Set `$GOPATH` variable to point to your Go workspace directory and add `$GOPATH/
   keys:<key:"app1" value:"test" >
   ```
 
+#### Verify key history
   ```
-  ./keytransparency-client history <email> --config=.keytransparency.yaml
+  keytransparency-client history <email> --config=.keytransparency.yaml
   Epoch |Timestamp                    |Profile
   4     |Mon Sep 12 22:23:54 UTC 2016 |keys:<key:"app1" value:"test" >
   ```
 
 
-## Running a Key Transparency Cluster
-
-1. Install 
-- [Etcd v3.1.3](https://github.com/coreos/etcd/releases/tag/v3.1.3).
-- [OpenSSL](https://www.openssl.org/community/binaries.html)
-- [Docker](https://docs.docker.com/engine/installation/) 
-  - Docker Engine 1.13.0+ `docker version -f '{{.Server.APIVersion}}'`
-  - Docker Compose 1.11.0+ `docker-compose --version`
-
-2. Install Key Transparency
-
-  ```sh
-  go get -u github.com/google/keytransparency/...
-  go get -u github.com/google/trillian/...
-  ```
-
-3. Run server setup 
-
-  ```sh
-./scripts/prepare_server.sh
-  ```
+## Running the server
 
-  The tool will build the server binaries, generate keys, and configure the server.
-  Clients will need the following public keys in order to verify server responses:
+### Install 
+1. [OpenSSL](https://www.openssl.org/community/binaries.html)
+1. [Docker](https://docs.docker.com/engine/installation/) 
+   - Docker Engine 1.13.0+ `docker version -f '{{.Server.APIVersion}}'`
+   - Docker Compose 1.11.0+ `docker-compose --version`
```

Comment:
Should we add `jq` as a dependency? (It is installed on travis by default and the script which is referred to for creating the trees will tell the user that he needs to install it anyways.)